visma/solvers/polynomial/roots.py

```diff
@@ -24,7 +24,15 @@
 
 
 def getRoots(coeffs):
+    '''Applies Quadratic Formula (https://en.wikipedia.org/wiki/Quadratic_formula) on the coefficients
+    of the quadratic equation
 
+    Arguments:
+    coeffs {list} -- list of coefficients of the quadratic equation
```

Comment:
```suggestion
        coeffs {list} -- list of coefficients of the quadratic equation
```

Same in line 34 and for docstrings in `quadraticRoots` and `findQuadraticRoots`